elements/campaign-display/components/campaign-display-image.js

```diff
@@ -3,13 +3,14 @@ import React, { PropTypes } from 'react';
 export default function CampaignDisplayImage (props) {
   return (
     <div className='campaign-display-image'>
-      <a href={props.url}>
-        <img src={props.image} />
+      <a href={props.clickthrough_url}>
```

Comment:
Something I noticed in avclub, we might need additional attributes on this anchor tag, like stuff for the analytics manager. Is there some way to we can pass arbitrary attributes to this anchor?

If not, or it's too sloppy, I think we can refactor out any issues the analytics manager has with resolving its attributes if those attributes don't live on the elements themselves.